tests/performance/ecommerce_hybrid_search/dataprep/generate_feed_query.py

```diff
@@ -367,17 +383,19 @@ def main():
         logging.info("Preparing query files...")
         shorthand_queries = human_readable_number(num_queries)
         for query_type in QueryTypeEnum:
-            vespa_save_path = (
-                Path(FINAL_DIR)
-                / f"vespa_queries-{query_type}-{shorthand_queries}.json.zst"
-            )
-            logging.info(f"Generating {query_type} queries")
-            save_vespa_query_files_from_df(df, vespa_save_path, num_queries, query_type)
-            es_save_path = (
-                Path(FINAL_DIR)
-                / f"es_queries-{query_type}-{shorthand_queries}.json.zst"
-            )
-            save_es_query_files_from_df(df, es_save_path, num_queries, query_type)
+            for category_filter in [False, True]:
+                filter = "-filter" if category_filter else ""
```

Comment:
consider using a different variable name than `filter` as it is a built-in function

Reply:
Fixed 👍